lib/fizzy/instantiate.cpp

```diff
@@ -535,7 +539,8 @@ std::optional<ExternalTable> find_exported_table(Instance& instance, std::string
     return ExternalTable{instance.table.get(), instance.table_limits};
```

Comment:
Can `.get()` not throw?

Reply:
It's `noexcept` https://en.cppreference.com/w/cpp/memory/unique_ptr/get